libs/gltfio/src/AssetLoader.cpp

```diff
@@ -288,9 +288,17 @@ void FAssetLoader::createEntity(const cgltf_node* node, Entity parent) {
     mResult->mEntities.push_back(entity);
     mResult->mNodeMap[node] = entity;
 
+
+    // Create a name component using the node label if it exists, otherwise use the mesh label. use default name if both empty.
+    const char* name = node->name ? node->name : (node->mesh && node->mesh->name)? node->mesh->name : "node";
```

Comment:
Please split this into two lines.

Reply:
* Split the name choosing logic
* Remove name creation in createLight() and union the name creation